javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -2345,7 +2393,7 @@ public static boolean sameTree(ExpressionTree expr1, ExpressionTree expr2) {
    * @return true if {@code caseTree} is the default case for a switch statement or expression
    */
   public static boolean isDefaultCaseTree(CaseTree caseTree) {
```

Comment:
The default case is also allowed to be written as:
```java
case null, default
```

Please discuss in the spec how this method behaves for such a case tree.